status-im/status-react
# feature #5582 - added BRLN coin (mainnet)

fixes #5582

### Summary:

Adds BRLN (Berlin Coin) token 

### Testing notes (optional):
Ping me for test coins

### Steps to test:
- Open Status in mainnet
- enable BRLN coin
- send, receive, check history, etc.

status: ready
